src/js/components/Banner.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Banner = props => {
+  return (
+    <div className={"banner-container message message-warning flush-bottom"}>
```

Comment:
Minor but why not only as string `"banner-container message message-warning flush-bottom"`

Reply:
Yes I noticed this too, I'll change it 👍